packages/aws-cdk-lib/aws-sqs/lib/queue.ts

```diff
@@ -195,6 +202,32 @@ export interface DeadLetterQueue {
   readonly maxReceiveCount: number;
 }
 
+/**
+ * Permission settings for the dead letter source queue
+ */
+export interface SourceQueuePermission {
+  /**
+   * Permission settings for source queues that can designate this queue as their dead letter queue
+   *
+   * @default - `RedrivePermission.BY_QUEUE` if `sourceQueues` is specified; otherwise, `RedrivePermission.ALLOW_ALL`.
```

Comment:
```suggestion
   * Permission settings for source queues that can designate this queue as their dead-letter queue.
   *
   * @default - `RedrivePermission.BY_QUEUE` if `sourceQueues` is specified,`RedrivePermission.ALLOW_ALL` otherwise.
```